docs/machine-learning/tutorials/image-classification.md

```diff
@@ -109,6 +110,7 @@ teddy4.jpg	toy
 toaster.jpg	appliance
 toaster2.png	appliance
 ```
+<!-- markdownlint-enble MD010 -->
```

Comment:
is this the right syntax @nschonni? Or should it be enable?

Reply:
thanks, that's a typo. I could leave off re-abling for the rest of the file, but I figure it's better to be explicit. I'll rebase this now

Reply:
fixed

Reply:
Thanks. Since this is inside the ML.NET content, I'll leave it up to that team to approve this one.